### DISCUSSION TITLE
Reality竟然可以用已过期的证书……

### DISCUSSION THREAD

**Discussion by Benjamin1919** [XTLS/Xray-core#2646] [General]
服务端：xray 1.8.4
客户端：passwall 2
配置：reality“自己偷自己”，模板参考https://github.com/XTLS/Xray-core/discussions/2246

现象：xray可以正常上网，但是伪装站打开提示这个：
<img width="1018" alt="截屏2023-10-17 18 08 48" src="https://github.com/XTLS/Xray-core/assets/72666597/be9a2066-62a7-4836-96ac-c0e1bc65f145">
这明显就是证书过期了，于是SSH连上VPS一看，证书都过期好几天了。

我记得以前单独用vision时，证书一过期，无论是xray还是伪装站全都连不上；
现在用了reality，即便证书过期也依然可以正常上网，但此时GFW如果进行主动探测的话肯定会知道这个证书过期了，这会不会成为一种特征？

**Comment by iKira** [comment]
xray用了Cloudflare的中间证书？

**Reply by Benjamin1919** [reply]
是的，acme以DNS方式申请，用的是zerossl的证书，有效期90天
